src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -0,0 +1,1160 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import com.google.common.annotations.VisibleForTesting;
+import htsjdk.samtools.*;
+import org.apache.spark.HashPartitioner;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.api.java.function.VoidFunction;
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.SparkProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.spark.utils.HopscotchHashSet;
+import org.broadinstitute.hellbender.tools.spark.utils.MapPartitioner;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+import scala.Tuple2;
+
+import java.io.*;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.function.Function;
+
+/**
+ * Tool to describe reads that support a hypothesis of a genomic breakpoint.
+ */
+@CommandLineProgramProperties(summary="Find reads that evidence breakpoints.",
+        oneLineSummary="Dump FASTQs for local assembly of putative genomic breakpoints.",
+        programGroup = SparkProgramGroup.class)
+public final class FindBreakpointEvidenceSpark extends GATKSparkTool {
+    private static final long serialVersionUID = 1L;
+    private static final int MIN_MAPQ = 20;
+    private static final int MIN_MATCH_LEN = 45; // minimum length of matched portion of an interesting alignment
```

Comment:
There are definitely examples of cases where you can get a high MAPQ on a weak but unique alignment. We encountered this with the bacterial contamination issue. We use OverclippedReadFilter to filter these guys out -- we could apply that here if we want to.